apps/financial-aid/backend/src/app/modules/aid/aid.service.ts

```diff
@@ -17,6 +18,23 @@ export class AidService {
     private readonly aidModel: typeof AidModel,
   ) {}
 
+  async create(aid: CreateAidDto, t: Transaction): Promise<AidModel> {
+    this.logger.debug(`Create aid or return existing one`)
+
+    const doesExists = await this.aidModel.findOne({
+      where: {
+        municipalityId: aid.municipalityId,
+      },
+      transaction: t,
+    })
+
+    if (doesExists) {
+      return doesExists
+    }
```

Comment:
Does this ever happen? Should municipalityId with type also be unique in this table?

Reply:
It can not be unique since we always have to make two rows one for each aid type 💸

Reply:
But no this should never happen really, I added it just in case, I dont know maybe remove it?

Reply:
Yes i am talking about the the rows together are unique, so you have
```
municipalityId : 10, type: individual
municipalityId : 10, type: cohabation
```
but you can't have:
```
municipalityId : 10, type: individual
municipalityId : 10, type: individual
```